### DISCUSSION TITLE
How to make markdown render disabled by default

### DISCUSSION THREAD

**Discussion by zvrkSam** [LazyVim/LazyVim#4185] [Q&A]
Before `markdown.nvim` was added i was using this code:
```lua
vim.api.nvim_create_autocmd({ "FileType" }, {
  pattern = { "markdown" },
  callback = function()
    vim.wo.conceallevel = 0
  end,
})
```
To have conceal be turned off by default and I could see all the links. Now `markdown.nvim` overrides this behavior and puts conceallevel to 1. Even if you turn off render markdown the conceallevel is always 1.

My question is if there is a way to have `markdown.nvim` turned off by default, so I can toggle it when I want, something similar in the code above.
I know you can disable it completely but I want to give this plugin a try.

**Comment by santised** [comment]
Yeah you can simply use the commands listed on their [github page](https://github.com/MeanderingProgrammer/markdown.nvim): try `:RenderMarkdown toggle`. 

Hope that helps.

**Reply by Flimm** [reply]
You can also use `:RenderMarkdown disable` if you prefer idempotent commands.

**Reply by Flimm** [reply]
I also found it helpful to run `:set conceallevel=0` to make Neovim display things that are concealed (for example: backticks surrounding inline code in Markdown, or link targets). By default, those things are concealed, except on the current line.

**Comment by zvrkSam** [comment]
Thanks I missed that part.
I will mark this as closed for the time being.
